amd64/crate/config/log4j2.properties

```diff
@@ -0,0 +1,60 @@
+# Crate uses log4j as internal logging abstraction.
```

Comment:
I am not sure whether there is a way around copying the assets to the folder with a docker file. These files must be a part of build context. I've checked some other projects and usually, the assets are also duplicated.